flare/gp_algebra.py

```diff
@@ -278,20 +278,19 @@ def get_neg_likelihood(hyps: np.ndarray, training_data: list,
 
     like = get_like_from_ky_mat(ky_mat, training_labels_np)
 
-    if monitor:
-        print('like: ' + str(like))
-        print('\n')
+    if output is not None:
+        output.write_to_log('like: ' + str(like)+'\n', name="hyps")
```

Comment:
A suggestion which is somewhat to personal preference: I personally find formatting strings using
` 'like: {} \n'.format(like) `
to be more readable.
I know that you also just translated the existing line this way, but going forward we could do all of our string formatting this way.